orderbook/src/database/orders.rs

```diff
@@ -120,6 +129,10 @@ impl OrderStoring for Postgres {
             .bind(order.order_creation.partially_fillable)
             .bind(order.order_creation.signature.to_bytes().as_ref())
             .bind(DbSigningScheme::from(order.order_creation.signing_scheme))
+            // TODO - remove this in https://github.com/gnosis/gp-v2-services/issues/901
+            .bind(1)
```

Comment:
Here I would use our rust contract artefacts to derive the correct contract address.

Reply:
This is cool... but we still need to decide if this field is going to be an address or not. We can push this forward by voting on https://github.com/gnosis/gp-v2-services/pull/906#discussion_r678190192

Reply:
So, in order to make this method (`insert_order`) non-blocking on web3 calls I would propose we either store the contract address on the Postgres struct (will require changes to its new constructor in many places). Alternatively we could make some dynamic global constant that can be imported here and used. Not super keen on updating every occurrence of `Postgres::new` since there are many.

<img width="465" alt="Screenshot 2021-07-28 at 07 29 31" src="https://user-images.githubusercontent.com/11778116/127315195-301398ef-686a-4422-a728-3fe2c7bd25e4.png">

Reply:
cf #907

Reply:
How about putting the contract address on the order metadata struct? This way it can be populated when we also verify the signature (at which point we need access to the contract address for the domain separator).

Reply:
Yes. I have actually already started a PR addressing issue #900 which will make this trivial and no longer require #907.